new/PRS14232TaklaIyasusMoa.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14232TaklaIyasusMoa" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Takla ʾIyāsus Moʾa</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-05">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ኢያሱስ፡ ሞአ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla ʾIyāsus Moʾa</persName>
+                    <occupation type="ecclesiastic">ʿaqqābe saʿat</occupation>
+                    <floruit notBefore="1450" notAfter="1535">Succeeded <persName ref="PRS0000000000">Tasfā Giyorgis</persName>
+                        in <date when="1486">1486</date> as ʿaqqābe saʿat of <placeName ref="LOC2260DabraE"/>.</floruit>
```

Comment:
LOC2260DabraE stands for Dabra Ǝgziʾabǝḥer, a minor church within the Hayq Estifanos complex

We used to have LOC2288DabraH for DHE which was deleted when INS0327DHE was created. (it was later decided that LOC should not be deleted, but for now we only have INS for Dabra Hayq Estifanos)

You need to refer to INS0327DHE throughout

Reply:
OK. Thanks for clarification!